app/src/main/java/com/brandon/forzenbook/network/mocks/ForzenApiServiceMocks.kt

```diff
@@ -1,52 +1,73 @@
 package com.brandon.forzenbook.network.mocks
 
+import android.util.Log
 import com.brandon.forzenbook.network.CreateAccountRequest
 import com.brandon.forzenbook.network.ForzenApiService
 import com.brandon.forzenbook.network.LoginRequest
 import com.brandon.forzenbook.network.LoginResponse
 import okhttp3.ResponseBody
 import retrofit2.Response
+import java.sql.Date
 
 class ForzenApiServiceAlwaysSuccessMock : ForzenApiService {
 
     override suspend fun login(request: LoginRequest): Response<LoginResponse> {
-        println("Brandon_Test_Network Always Success Mock")
+        Log.e("Brandon_Test_Network","Always Success Mock")
         return Response.success(
             LoginResponse(
                 token = "ThisIsASixtyFour(64)CharacterLongStringForTestingNetworkResponse"
             )
         )
     }
 
-    override suspend fun createUser(request: CreateAccountRequest): Response<Any> {
-        return Response.success(200)
+    override suspend fun createUser(
+        firstname: String,
+        lastname: String,
+        dateOfBirth: Date,
+        location: String,
+        email: String
+    ): Response<Void> {
+        TODO("Not yet implemented")
     }
 
+
 }
 
 class ForzenApiServiceAlwaysFailsMock : ForzenApiService {
 
     override suspend fun login(request: LoginRequest): Response<LoginResponse> {
-        println("Brandon_Test_Network Always Fail Mock")
+        Log.e("Brandon_Test_Network","Always Fail Mock")
         return Response.error(404, ResponseBody.create(null, "Not Found."))
     }
 
-    override suspend fun createUser(request: CreateAccountRequest): Response<Any> {
-        return Response.error(404, ResponseBody.create(null, "Not Found."))
+    override suspend fun createUser(
+        firstname: String,
+        lastname: String,
+        dateOfBirth: Date,
+        location: String,
+        email: String
+    ): Response<Void> {
+        TODO("Not yet implemented")
```

Comment:
Same as other comment, ticket number for all TODOs

Reply:
# added